.circleci/config.yml

```diff
@@ -63,15 +64,49 @@ jobs:
           command: |
             ./.circleci/set-nameserver.sh sudo
             sudo apt-get install gettext-base
-      - attach_workspace:
-          at: ./data
+      - run:
+          command: |
+            sudo mkdir -p /cncf/data
+            sudo chown -R circleci:circleci /cncf/
+          name: Cache Prep
+      - restore_cache:
+          key: cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}
```

Comment:
This is restored from a CircleCI internal cache? Is it restoring a directory?

Reply:
It is restoring from the cache.  ```cncf-data-{{.Environment.CIRCLE_WORKFLOW_ID}}``` is the cache key to be restored.  It restores the directory originally cached, which was /cncf/data